spec.md

```diff
@@ -2692,6 +2692,15 @@ module MyModule :
    out <= r$0
 ```
 
+# Semantics of Values
+
+FIRRTL is defined for 2-state logic.  The behavior of a generated circuit in a
+language, such as Verilog or VHDL, which have multi-state logic, is undefined
+in the presence of values which are not 2-state.  A FIRRTL compiler need only
+respect the 2-state behavior of a circuit.  This is a limitation on the scope of
+what behavior is observable (e.g. a relaxation of the
```

Comment:
Should this be `i.e.` instead of `e.g.`?

I wonder how much the last sentence adds in general. It would be good to have a section on what it means for a firrtl compiler to "preserve semantics", but I recognize that would be a lot of work.